integration_tests/ctesque/src/androidTest/java/android/app/ContextTest.java

```diff
@@ -71,6 +72,7 @@
 @RunWith(AndroidJUnit4.class)
 public class ContextTest {
   private static final int APP_WIDGET_HOST_ID = 1;
+  private Context application;
```

Comment:
Maybe rename it `context`, since it is not an instance of the `Application` class?